test_integration/05_facilitator_stop/01_stop.test.ts

```diff
@@ -12,6 +12,6 @@ describe('facilitator stop', async (): Promise<void> => {
       facilitatorKill,
       { stdio: [process.stdout, process.stderr], env: process.env },
     );
-    fs.removeSync(Directory.getFacilitatorConfigPath(Constants.auxChainId.toString()));
+    fs.removeSync(Directory.getFacilitatorConfigPath(parseInt(Constants.auxChainId)));
```

Comment:
We should update Constants.auxChainId to be number. It's string now "1000".
After that we don't need to parseInt. Wherever this constants is called parseInt or parse to Number can be removed.